### DISCUSSION TITLE
How we do things around here (life-like Governance)

### DISCUSSION THREAD

**Discussion by dilgreen** [CredComSoc/ProtocolWorkingGroup#9] [General]
_Everything below perhaps needs to be prefaced - the contexts I am talking about are ones conditioned by voluntary, not coercive, engagement. Another important consideration here will 'forking' rights. Also, the process described here is not, by itself, considered  'sufficient', but a ground basis for participation._

Generally, people don't like to think about Governance, outside two contexts; 

1. conversations at the beginning of something (beginnings to include both 'founding' and 'joining' conditions),
2. when a controversy becomes troublesome.

In neither case, in my experience, do they actually _want_ to work on Governance, but rather get to an outcome.

In the first context, the amount of energy is limited. People have started something because they want to  pursue it, not to talk about what to do if it goes wrong. The outcome people want is to feel safe - typically in relation to some bad experience they have had/seen in the past and want to avoid.

In the second, people don't really want to work on Governance - the outcome they want is for the controversy to be resolved in a way which doesn't leave a bad feeling.

**If we accept this characterisation, and take it seriously, we see that what people want is to get on with the work they came to do, and feel safe doing so, without spending too much time on mechanics.** 

And this seems reasonable and healthy.

This led me to start thinking about what I call ‘life-like’ governance. What would it mean for one’s engagement with the governance of a project to feel like part of life?

I wrote about it [here](https://digital-anthropology.me/2018/04/06/life-like-governance/), but it can be boiled down to a recursive protocol, and a tiered approach to policy:
- **PROTOCOL ONE**: Do what you think comes next. 
    - Only if it occurs to you, or is represented to you, that what you are doing might not ‘fit’, do you need to consider  ‘policy’. If you do need to consider policy, go to Protocol Two.
- **PROTOCOL TWO**: Consider the appropriate Policy tier: Find one or more peers (if the problem was represented to you, include those peers) to discuss the issue in respect of the relevant Policy (a ‘quorum’ Policy may apply).
    - If, with them, you can find a way to do what you think comes next that resolves the issue, go to Protocol One (_you accept the Policy, and modify your intentions_).
   - If you cannot find a resolution, then you have two choices
        - ACCEPT- think of something else it makes sense to do, and revert to Protocol One (_on this occasion, you accept current Policy as a blocking constraint upon what you can do_).
        - SEEK CHANGE- gather enough of your peers (a ‘quorum’ Policy may apply), to develop, refine and propose either a new Policy or a change to the relevant Policy.
            - If the change you seek is constrained by Policy at a deeper Tier, recurse on Protocol Two.

No more than four ‘Policy’ tiers would seem necessary:
- Daily Life – getting on with the job,
- Operational policy – practical systems,
- Core policy – operationalising the Heart Story as a basis for systems design,
- Heart – the deep story of why any of us is here in the first place.

Essentially, this method proposes that there are no special people in control of more fundamental policies. Deeper policies simply have more friction about them - the reference to 'quorum' conditions is key. A member wanting to do something controversial will find that they have to convince more and more peers to dig down through the layers and spend time working on Policy, rather than getting on with the work.
Forking rights matter in this context, because inertia among peers is not necessarily a sign of 'correctness'.  If a member considers some shift important, but peers do not, then it may be time to fork.

**Comment by dilgreen** [comment]
I have another long post drafted, which is more specific, concerning relations with the Credit Commons Society.
But I think I'll wait a day or two to see if/how the above 'lands'.

**Comment by matslats** [comment]
Now I understand what you mean by 'life-like' governance.
I like it!

**Comment by JohnG-cc** [comment]
That sounds good @dilgreen .
It might take me a while to get the hang of it though, I've only really worked with "chain of command" or "consensus committee" before. This sounds a lot more fluid (which I think is a good thing).

**Reply by dilgreen** [reply]
I think we're already working like this, here. 

That's the aim of 'life-like <x>' approaches - as far as possible, they map onto healthy things ppl do without thinking about it.

I see people here putting ideas out, then carefully signalling - checking in about aspects of them which they think might be controversial.

This is Protocol One in action - Protocol One is mostly ethics/culture.

When someone thinks that the next thing they want to do is suggest that something which has been checked out for a while, and received no 'blocking' feedback, should be explicitly recognised, then we'll have our first Policy.

Then, if they get enough ppl to say yes - something that feels like a quorum (probably not a hard policy yet), and no-one blocking, then I guess they'll make a Policy document, and we'll be on our way - Protocol Two.

**Comment by dilgreen** [comment]
I guess I should attempt some mapping. The life-like Governance blog post was written long before the Direction, Vision thing was developed.
I think it goes like this:
The **Heart** of an organisation is the **Vision**.
**Direction** is a 'defining Ethic' - distinct from the more general **Ethics** which will be more meta.
**Policies** will be instantiated from **Ethics** in service of the **Heart**.
**Next Steps** will be **Daily Life**.

![DirectionVisionSteps-ethics](https://github.com/CredComSoc/ProtocolWorkingGroup/assets/5934679/2c6a154d-da95-48e2-bff6-c1aa6401bbad)

**Comment by dilgreen** [comment]
While the above is about our culture, this group exists in a context, which already has significant definition, and also some promise of a larger projected context which (at least) the current group here subscribes to.

In the future, if this work goes as we hope it will, people are likely to turn up here who do not subscribe fully to these wider contexts (existing or projected). That has to be OK - one of the reasons for deconstructing the catch-all 'purpose' thing is to make it easier for people who believe in a **Next-Step** to contribute, even if they don't share (or necessarily even care to ask about) the **Vision**.

The question is raised, by this future scenario, then, about the relationship of this project to that wider context - to its Direction, Vision, etc. For if we are successful, it becomes entirely possible - I would go as far as to say desirable - that there are many people here only interested in the economic utility that CCP/CredCom provides.

In such a condition, how do we ensure that some version of technocratic capture does not take this group onto a different course - one which we would find problematic?

And more, how can we act to keep this group 'on course' (our version of what that means, at least), without becoming overly controlling (which has other negative implications)?

Elinor Ostrom's work in 'Governing the Commons' (most clearly visible in her section on game theory) suggests a way through. The primary research part of the work then provides significant empirical evidence for the efficacy of that suggestion, which is that some aspects of governance are delegated to a 'monitor'  (I describe such a role as 'interested', but also 'disinterested' - deeply concerned, but without direct skin-in-the-game). Handily, this chimes with the VSM's system 5.

@tfwoodroofMCS and I currently label this role as '**Commons Safeguarder**' - which is clunky, but we don't like anything better yet (Guardian, Custodian, Arbiter, Monitor all come with baggage). Functionally, the role is threefold:
1. to care deeply about the Direction, the Ethics, and the developing Vision
2. to pay attention
3. to intervene only in 'you should think again' mode, when some proposal looks as if it will take the project off course.

Note that there is not even veto power here. However, the cultural weight of a 'think again' message (which of course needs clearly communicated reasoning), should be significant, if the group is healthy. Further, there is the possibility of being sanctioned within / excluded from wider contexts for which the Safeguarder provides confidence that the project is doing what the context needs it to.

For the purposes of this thread, this adds a 'special' peer into the mix - one which is increasingly involved as changes get closer to the Heart/Vision or Ethics/Direction. 

As to the wider context, that belongs in another thread, which I will make soon.
EDIT: [Here is that other discussion, concerning the relationship with the Credit Commons Society](https://github.com/CredComSoc/ProtocolWorkingGroup/discussions/11)

**Comment by dilgreen** [comment]
Here's a nice, non controversial training wheels issue to play with.

I've used CCP as an TLA for Credit Commons Protocol in several places.

@JohnG-cc commented that it is not a fortunate abbreviation, since the standard connotation is the Chinese Communist Party (who I somehow doubt will be aligned).

In other places, I have used CredCom - which is short enough to type, and less likely to cause confusion.

However, consistency seems desirable, so I'm going to make a proposal for a Policy (at the Operational tier) - that we adopt CredCom as the official abbreviation for The Credit Commons Protocol.

Let's see if this gets any responses.

I guess a thumbs up is a consent, a question is a request for clarification, and no response is 'don't care'.

I have no idea what the 'quorum' is, but this is the most superficial tier of policy, so maybe if 50% respond, we've got something to work with?

**Reply by JohnG-cc** [reply]
👍

**Reply by utunga** [reply]
👍 

Works for me - CredCom as the preferred short hand for Credit Commons or Credit Commons Protocol..

**Reply by tfwoodroofMCS** [reply]
👍 Fine with me, although I'd prefer it to refer only to the Protocol. From the [Protocol page](https://www.mutualcredit.services/credit-commons-protocol) on MCS' website:

> This fractal structure [of the Protocol] mirrors that of sociocracy, suggesting a natural fit between the resulting economic and participative governance structures. Trade and decision-making is supported across and between networks, enabling scale and diversity of economic activity without the requirement for centralisation. Groups can therefore remain small, trust-based, and self-governing within a larger, decentralised, yet complex economy of similarly autonomous actors that is referred to as the ‘Credit Commons’.

I wrote this because some statement of the relationship between the two seemed necessary, but I'm not suggesting it is in any way canonical (aside from anything else, it's not nearly concise enough to serve as a working definition!). However, I think the distinction needs to be maintained.

**Reply by dilgreen** [reply]
OK, so that looks as if 80% of us engaged with the proposal, positively. so that looks as if it is adopted.
I suppose that the next question is - where is it to be documented?

A separate thread, methinks.

But CredCom  it is (until someone suggests to reviewing it, and gets enough interest: Note: Sociocracy also recommends setting a review date at the point of decision. Somehow that doesn't feel so appropriate here? Small group, high level of thoughtful enquiry, agile approach by default, inherently meta-stuff going on here?)

**Reply by tfwoodroofMCS** [reply]
I think sociocracy-style reviews could be a way of operationalising the gradual stiffening pattern, and provide ways for relatively recent contributors to get a sense of how the deeper policy tiers evolved. However, I can see what you're saying @dilgreen that reviewing *every* policy may not be appropriate.

A proposal (which follows on from my separate, earlier response below):
- 'Daily Life' policies (such as the 'Protocols 1 + 2' approach) are only subject to formal review if this is explicitly agreed to be necessary - otherwise, we assume that the careful attention of the people doing the work, and the agile approach, will provide sufficient feedback loops. However, such policies should still be explicitly articulated/documented for the sake of shared learning as they emerge/are realised.
- As 'Core' and 'Operational' policy tiers develop in response to 'Daily Life', these are likewise documented and *are* subject to review, with timescales decided on an ad-hoc basis as seems appropriate.
- 'Heart' policies are also subject to review, but over a significantly longer timescale than is typical for the 'Core' and 'Operational' tiers, or if it becomes obvious that there is a divergence (emerging or previous undiscovered) between people's deep reasons for being here.

Since this seems like a 'systems design' policy, I would place it at the 'Core' tier, which suggests that - if adopted - it should, by its own criteria, be subject to review ;)

With respect to the original question around referring to the Protocol as 'CredCom', I would categorise this as 'Daily Life', and therefore not subject to review unless someone explicitly raises it as requiring further discussion.

**Comment by tfwoodroofMCS** [comment]
Since #2 also referenced the 'gradual stiffening' pattern as applicable to Visions, and a Vision is a thing that is only buildable with significant ambition and coordination (the overheads of which policies are meant to reduce), it seems worth exploring how development of 'policies' across 'tiers' might be understood in this way:

- Establishing a shared 'Protocols 1 and 2 + Tiers' meta-understanding seems to me like a sufficiently global yet 'flimsy' structure for this project, given that it already sits within the wider context outlined in #11.

- 'Deeper' policies (closer to the 'Heart' tier) are intended to have increasing amounts of 'friction', which seems to map pretty clearly onto 'stiffness'. This suggests that these should be outlined early on, but adhered to - at least initially - primarily in spirit rather than letter. This makes sense to me for a few reasons:
   - In the early stages, it's likely that the only people present will be those who care enough to potentially become highly dedicated, but they will want to establish a good level of confidence that others are aligned enough to justify the commitment.
   - Even for these people, their 'deep story' for being there may change as they understand different facets of the work (this has happened for me).
   - There are a lot of projects competing for attention, and being able to articulate non-superficial motives is an advantage.

- The risk here is that these early contributors become by default the 'special people in control of more fundamental policies'. The 'gradual' element of the pattern seems to serve as risk-mitigation (along with the quorum requirement, sociocratic processes, and indeed the explicit discussion here).

- Developing the 'Core' and 'Operational' policy tiers first and foremost in response to 'Daily Life' activities is what I understand the 'Protocols 1 and 2' suggestion to achieve, and I hope that - assuming they will be largely accepted on the basis of ability to fairly rapidly participate in the daily life of getting on with the work - new contributors will soon feel confident to engage with and stiffen these deeper tiers if they wish to.

- However, systematising these tiers may require a separate set of competencies, again introducing a risk of 'special people' who end up stiffening the structure in ways that are not fully understood or accepted by others. I imagine this could be one source of the 'technocratic capture' in @dilgreen's more recent comment, mitigated by a Commons Safeguarder.

**Reply by dilgreen** [reply]
The above is excellent. Thanks @tfwoodroofMCS.

We need an actionable Next Step on this - not that it's a critical path item, but would be good to get a thread going.

**Comment by dilgreen** [comment]
OK, here's the threatened thread on 'where do we record decisions on _How we do things around here?_'

Since this is a repo, an obvious answer would seem to be a doc here, with commits!

Could start out as simply numbered items points in  prepend log style, with some minimal syntax (such as `index number : date : <supersedes index x> : policy text : discussion link`) ??

**Reply by dilgreen** [reply]
But also worth thinking about, perhaps, since the CredCom Society (CSS) is looking for a way to implement a '[logging](https://patterns.sociocracy30.org/logbook.html)' process for all its 'Circles', I wonder, if there is a simple editor which non tech ppl might find manageable, that would enable this functionality for all, in another repo under CredComSoc?

I further wonder, whether a simply structured GitHub Wiki might help ?

Before this looks like a suggestion that this Protocol Working Group start building products for CredComSoc, I'm definitely not suggesting that. 

But there are people wanting to help with this project, who might be interested in doing such work, with the help of some requirements.

**Comment by dilgreen** [comment]
**PROPOSAL**: After various conversations, I hope this is a non-controversial suggestion. But it is not an inconsequential one, so open discussion is desirable, with the aim of improving/refining the proposal (or finding an alternate way forward).

**ISSUE**: There is a disjunction between two important strands in the group, which need to be understood, accepted, and resolved:
1. @matslats has a current work-stream for CredComs. 
    - This is active work, with significant constraints - not least that Matt is working alone on a significant Drupal/PHP codebase, with specific userbase in mind (hundreds of  HamLets user communities, which he intends to move across to CredCom-based system)
3. This group exists to work together to build a version of CredCom which incorporates the best thinking this group can bring to bear, in the context of a broad vision for the value CredCom could offer.
    - This is careful, steady work, looking to develop something robust and generic - taking the Credit Commons whitepaper, and learnings from the existing versions of CredCom as the basis for development, but not wishing to be unduly constrained.

**THE PROPOSAL: That this group identifies, and recognises the different conditions of production of, two 'tracks' for CredCom, as follows:**

- **TRACK ONE** - working toward Version 1 of the Credit Commons Protocol
   - @matslats has 'product owner' status within this track.
   - This release will be supported by this group.
 - **TRACK TWO** - working towards a Version 2 of the Credit Commons Protocol
     - This group, working under its governance processes, is in control of this track.

The intent of this proposition is to:
1. Enable @matslats to continue to progress CredCom within this group, without differences between choices he makes, and the outcomes of this group's work creating tension.
   5. Thus Matt , as 'product owner' of Track One, can make the decisions he feels appropriate in respect of his focus on the needs of the existing user groups he serves, and so on (see points below).
      - Such decisions, as appropriate,  should be made in the context of discussions in this group (even if this group's best thinking suggests a different course of action from the one Matt takes, it is important for that difference to be both understood and visible).
7. Allow this group to work on a version of the protocol which incorporates the best thinking of the group as a whole, with @matslats a full participant in the group.

A concern arising, clearly, is the approach taken to future compatibility. Any CredCom Version 2.x developed in this way would need to consider ease of adoption by existing projects which have been build using Version 0.x/1.x.

The floor is open! We need to hear from everyone, on this one, I think...

**Reply by matslats** [reply]
I suggest a more flexible approach.
We're talking about a roadmap here.
We can lay out the milestones in advance.
I'm hoping to deliver version 0.2 by mid July. 
Any proposal for improvements can be given a number which reflects its importance and its difficulty.
Therefore I suggest
Version 1.0 as a 'complete' version of the protocol - the 'diamond'
Version 2.0 as a high volume version of the protocol - the 'tiara', perhaps?
And intermediate numbers can be made as we approach them

**Reply by tfwoodroofMCS** [reply]
I need to preface my reply by saying that I am not going to be doing *any* of the technical work here, as I am not a coder. My understanding of the requirements for CredCom comes entirely from semi-regular conversations with @dilgreen over the last couple of years, almost always from the perspectives of:

- The extent to which recursive mutual credit ledgers in general can be expected to map onto real-world economic structures (#6)
- The implications of CredCom's development status (#3) with respect to MCS' projects & overall strategy
- How and where its development should be supported (#11).

With that out of the way, I'm strongly inclined to support @dilgreen's proposal, as it explicitly lays out an understanding that has clear implications for who might do what in the near term, and how these activities will simultaneously support both present/user-oriented continuity and necessary future-oriented 'foundational' work, without a risk of crossing wires.

I would like to ask @matslats for a clarification, though: when you refer to Version 1.0 as 'complete', do you mean that it incorporates 'the best [broad] thinking this group can bring to bear' (as @dilgreen described with respect to Track Two)?

If so, does this not suggest your working on Version 0.2 is essentially @dilgreen's Track One, with Track Two being the work by this group necessary to make Version 1.0 'complete'?

**Reply by JohnG-cc** [reply]
@tfwoodroofMCS @matslats @dilgreen
> If so, does this not suggest your working on Version 0.2 is essentially @dilgreen's Track One, with Track Two being the work by this group necessary to make Version 1.0 'complete'?

Edit: never mind, I had misunderstood @matslats' 0.2/1.0/2.0 layers...

**Comment by utunga** [comment]
Just on voting mechanics.. since LoomIO built something for this  purpose of taking non-restrictive votes on things does it make sense to reinvent the wheel? I know there'd be a certain amount of friction introducing a new platform but if we intend to make lots of decisions?
